src/Andoromeda.Kyubey.Models/Token.cs

```diff
@@ -33,12 +33,12 @@ public class Token
         [MaxLength(16)]
         public string Id { get; set; }
 
-        public byte[] Icon { get; set; }
+        //public byte[] Icon { get; set; }
 
         [MaxLength(32)]
         public string Name { get; set; }
 
-        public string Description { get; set; }
+        //public string Description { get; set; }
 
         public long? DeliveryAmount { get; set; }
```

Comment:
Which place the DeliveryAmount has been used?

Reply:
This field is still in use